lib/ansible/plugins/lookup/fileglob.py

```diff
@@ -18,6 +18,7 @@
         required: True
     notes:
       - Patterns are only supported on files, not directory/paths.
+      - file lookup occurs relative to one or more files/ folder(s) when relative paths are specified.
```

Comment:
not really, this is only part of it, a link to 'ansible path searching page' might be a better option https://docs.ansible.com/ansible/latest/user_guide/playbook_pathing.html#task-paths